examples/firebase-auth-firestore/app/routes/index.tsx

```diff
@@ -34,18 +34,21 @@ export type ActionData = {
 export const action: ActionFunction = async ({ request }) => {
   const { uid } = await requireAuth(request);
   const form = await request.formData();
-  if (request.method === "POST") {
+  const action = form.get("action");
+  if (action === "create") {
     const title = form.get("title");
-    if (typeof title !== "string")
+    if (typeof title !== "string" || !title.length) {
```

Comment:
```suggestion
    if (typeof title !== "string" || title.length === 0) {
```

Reply:
Thanks, is this for performance, personal preference or consistency with the rest of the code base?

Reply:
It makes it more clear what you're actually checking.